### ISSUE TITLE
Generated typst doesn't compile

### ISSUE THREAD

**Issue by jscarrott** [LegNeato/mdbook-typst#17]
Just tried this project out and when I try to compile the typst I get:
`error: unknown variable: row
    ┌─ book/typst/book.typst:310:1
    │
310 │ #row[
    │  ^^^`
    
In generated tables? I'm not super familiar with typst but can't find any reference to #row.

**Comment by LegNeato** [comment]
Do you happen to have a public link to the book that is causing the issue?

**Comment by LegNeato** [comment]
Yeah, not sure where that came from...looks like tables are currently busted. It will take a bit more work as the column number is required for typst but markdown doesn't give us that by default.

(note to self, docs on table: https://typst.app/docs/guides/table-guide/ )

**Comment by Atreyagaurav** [comment]
The table syntax seems to be changed, you can fix it by removing the table rows/cell and following the new syntax.

Any updates on this? Manually changing a lot of tables and other things is a hassle.

**Comment by LegNeato** [comment]
This should be fixed on 0.1.8, which was just pushed.

**Comment by Atreyagaurav** [comment]
Thanks, I ended up writing my own lol: https://github.com/Atreyagaurav/mdbook-nadi-typst

It doesn't have a lot of features, but it works for my use case.
